virttest/utils_misc.py

```diff
@@ -2374,8 +2377,10 @@ def compare_qemu_version(major, minor, update, is_rhev=True, params={}):
     if installed_version is None:
         logging.error("Cannot get local qemu version, return False directly.")
         return False
-    if is_rhev != installed_version['is_rhev']:
-        return False
+    if is_rhev or is_ma:
+        if not ((is_rhev and installed_version['is_rhev']) or
+                (is_ma and installed_version['is_ma'])):
+            return False
```

Comment:
when we use qemu-kvm that is not part of RHEV ie not ma and not rhev, then this API should not return false directly which is inconsistent.
let us have is_rhev and is_ma param in compare_qemu_version() as False by default,
`def compare_qemu_version(major, minor, update, is_rhev=False, is_ma=False, params={}):`

so that, compared qemu even if it is not part of rhev can be compared appropriately. The calling function should set the param is_rhev or is_ma for comparing.